ui/pages/create-account/connect-hardware/index.scss

```
@@ -124,7 +124,7 @@
   }
 
   &__error {
-    color: #f7861c;
+    color: var(--color-secondary-default);
```

If it's an error we could just use the error color?

```suggestion
    color: var(--color-error-default);
```